spec/widgets/widgets-collection.spec.js

```diff
@@ -12,27 +12,12 @@ describe('widgets/widgets-collection', function () {
     this.collection = new WidgetsCollection();
     spyOn(this.collection, 'sort');
     this.collection.reset([
-      {order: 0, isColorsApplied: false},
+      {order: 0, autoStyle: false},
       {order: 1},
-      {order: 2, isColorsApplied: true}
+      {order: 2, autoStyle: true}
     ]);
   });
 
-  it('should only allow applying colors in one widget at a time', function () {
```

Comment:
Ok, then what is [this](https://github.com/CartoDB/deep-insights.js/pull/304/files#diff-8165807a4462edc13831b72bc1328180L18) doing?